core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -964,6 +974,8 @@ public function copyAll()
 		{
 			foreach ($arrClipboard[$this->strTable]['id'] as $id)
 			{
+				// No need for permission check here, it's done in copy()
```

Comment:
should try-catch and skip if a `copy` operation is denied

Reply:
Done in 596477a7322a89937cf775075b03c7521be9d120.